src/test/java/org/openmrs/contrib/qaframework/page/PatientVisitsDashboardPage.java

```diff
@@ -45,8 +50,28 @@ public String getPageUrl() {
 		return "coreapps/patientdashboard/patientDashboard.page";
 	}
 
-	public void goToCaptureVitals() {
-		findElement(CAPTURE_VITALS).click();
+	public PatientCaptureVitalsPage goToPatientCaptureVitalsPage() {
+		waiter.until(ExpectedConditions.presenceOfElementLocated(CAPTURE_VITALS));
```

Comment:
does the element `CAPTURE_VITALS` take long to be loaded on the page?

Reply:
@kdaud , I don't think it does since it is  one of the major elements on the Patient visits dashboard page.

Reply:
so any reason for adding a waiter call to the element?